src/github.com/matrix-org/rageshake/submit.go

```diff
@@ -93,12 +95,20 @@ func (s *submitServer) saveReport(ctx context.Context, p payload) error {
 	//  "bugreport-20170115-112233-N.log.gz" => oldest log
 	t := time.Now().UTC()
 	prefix := t.Format("2006-01-02/150405")
-	summary := fmt.Sprintf(
-		"%s\n\nNumber of logs: %d\nVersion: %s\nUser-Agent: %s\n", p.Text, len(p.Logs), p.Version, p.UserAgent,
+
+	var summaryBuf bytes.Buffer
+	fmt.Fprintf(
+		&summaryBuf,
+		"%s\n\nNumber of logs: %d\nApplication: %s\nVersion: %s\nUser-Agent: %s\n",
+		p.Text, len(p.Logs), p.AppName, p.Version, p.UserAgent,
 	)
-	if err := gzipAndSave([]byte(summary), prefix, "details.log.gz"); err != nil {
+	for k, v := range p.Data {
+		fmt.Fprintf(&summaryBuf, "%s: %s\n", k, v)
```

Comment:
Assuming https://github.com/matrix-org/rageshake/pull/6/files#r110698886 is true, you'll need to change this too.